site/content/internal/onboarding/manager-guide.md

```diff
@@ -32,4 +32,10 @@ weight: 30
 - Add new staff to the group
 
 ### Core Committer Mug
-- Message @lindy and ask her to send the new hire their core committer mug
+- Message @lindy.isherwood and ask her to send the new hire their core committer mug
```

Comment:
this is now done by @hannaparks apparently

Reply:
```suggestion
- Message @hanna.park and ask her to send the new hire their core committer mug
```